broker/broker-route-manager/src/main/java/org/codice/ddf/broker/route/manager/DynamicRouteDeployer.java

```diff
@@ -32,15 +32,34 @@
 public class DynamicRouteDeployer implements ArtifactInstaller {
   private static final Logger LOGGER = LoggerFactory.getLogger(DynamicRouteDeployer.class);
 
-  private final CamelContext camelContext;
+  private static Map<File, List<RouteDefinition>> processedFiles = new ConcurrentHashMap<>();
 
-  private Map<File, List<RouteDefinition>> processedFiles = new HashMap<>();
+  private final CamelContext camelContext;
 
   public DynamicRouteDeployer(CamelContext camelContext) {
     this.camelContext = camelContext;
   }
 
-  public void init() {}
+  public void init() throws Exception {
+    if (processedFiles.isEmpty()) {
+      return;
+    }
+    processedFiles
+        .entrySet()
+        .stream()
+        .forEach(
```

Comment:
❗️ Replace the double streaming with `flatMap()`. For example: ```processedFiles.values().stream().flatMap(List::stream).forEach(routeDefinition -> ...)```

Reply:
I am calling camelContext.addRouteDefinitions(...) instead, so we don't need a flat map anymore.